### PR TITLE
Change references to the Google Groups to point to the forum instead

### PR DESCRIPTION
**Pull request by wetneb**
We want to migrate out of the Google Groups and use [the Discourse forum](https://forum.openrefine.org/) instead.

Some links in versioned documentation were left as is. I plan to merge all versions of the versioned docs together in OpenRefine/openrefine.github.com#140.

### CODE REVIEW

**Comment by wetneb** [comment]
Merging soon unless there are concerns.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -29,7 +29,7 @@ You can contribute code in various ways:
 
 All developers including new distributions and plugin developers are invited to leverage the following OpenRefine project management areas to avoid splitting the community in different communication channels.
 - the [official documentation](https://docs.openrefine.org/technical-reference/technical-reference-index) for shared documentation between both user docs and [technical reference](https://docs.openrefine.org/technical-reference/technical-reference-index)
-- the [developer mailing list](https://groups.google.com/forum/?fromgroups#!forum/openrefine-dev) for technical questions, new feature development and anything code related. We invite you to share you idea first via the developer mailing list. Someone may be able to point out to existing development saving you hours of research and development. 
+- the [developer forum](https://forum.openrefine.org/c/dev/8) for technical questions, new feature development and anything code related. We invite you to share you idea first via the developer mailing list. Someone may be able to point out to existing development saving you hours of research and development.
```
Comment (tfmorris) [inline CONTRIBUTING.md:32]: This still has a reference to the mailing list. Presumably it needs to mention some tag or category or whatever the appropriate thing in forum lingo is.

**Review by tfmorris — COMMENTED** [review]
I was kind of skimming by the end, so might have missed some stuff, but noticed a couple of little things that might need tweaking

**Comment by wetneb** [comment]
Since of course I forgot to merge this before migrating the docs to the other repository, I will close this PR and integrate the changes (including the suggestions by @tfmorris) directly in the other repository.

File: docs/docs/manual/troubleshooting.md [hunk 2]
```diff
@@ -13,19 +13,17 @@ If you don’t find your problem and solution there, continue on to the resource
...
 
 ### If you’re having a problem: {#if-youre-having-a-problem}
-*   Search the [User forum](https://groups.google.com/g/openrefine) to see if the problem is already reported
+*   Search the [forum](https://forum.openrefine.org) to see if the problem is already reported
 *   Search [Github issues](https://github.com/OpenRefine/OpenRefine/issues) to see if the problem is already reported
 *   Read [Stack Overflow](https://stackoverflow.com/questions/tagged/openrefine) to see if others had a similar problem
 *   Check [Twitter](https://twitter.com/search?f=tweets&vertical=default&q=OpenRefine%20OR%20%22Open%20Refine%22%20OR%20%23OpenRefine&src=typd) to see if others are discussing the problem
 *   Report an issue: 
-    *   First as a new thread (conversation) in the [User forum](https://groups.google.com/g/openrefine).
+    *   First as a new thread (conversation) in the [forum](https://forum.openrefine.org/).
```
[the remark this answers is not in the document]
Comment (wetneb) [reply docs/docs/manual/troubleshooting.md:21]: I think it is also fine to report bugs on the forum if people are intimidated by or do not have an account on GitHub. The mailing list often served as first triage area before the issue tracker, so I would assume the forum can still serve this purpose. I am rephrasing that over in the website repository.

**Pull request merged**